java/arcs/core/allocator/Allocator.kt

```diff
@@ -70,7 +70,11 @@ class Allocator(val hostRegistry: HostRegistry) {
     suspend fun lookupArcHost(arcHost: String) =
         hostRegistry.availableArcHosts().filter { it ->
             it.hostId == arcHost
-        }.firstOrNull() ?: throw ArcHostNotFoundException(arcHost)
+        }.firstOrNull() ?: throwError(arcHost)
+
+    fun throwError(arcHost: String): Nothing {
+        throw ArcHostNotFoundException(arcHost)
+    }
```

Comment:
Can this be private?

Reply:
I can inline it, IntelliJ, for whatever reason, has issues debugging coroutines and sometimes I have to break things out into a normal function to breakpoint them, and forgot to put it back. :(